File: Data Structures & Algorithms/products-of-array-discluding-self/submission-8.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {

        vector<int> res(nums.size());

        for(int i = 0; nums.size() > i; i++){ 
            int total = 1;
            for(int j = 0; nums.size() > j; j++){
                if (i !=j){
                total *= nums[j];
            }
            }
            res[i] = total;
        }

        return res;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-8.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:


        map = {}

        for i, num in enumerate(nums):
            value = target - num
            if value in map:
                return [map[value], i]
            map[num] = i



        return false
            

            



        